src/applications/check-in/day-of/pages/CheckIn/DisplayMultipleAppointments.jsx

```diff
@@ -60,13 +60,13 @@ const DisplayMultipleAppointments = props => {
         {format(new Date(), "MMMM d, yyyy 'at' h:mm aaaa")}
       </p>
       <p data-testid="refresh-link">
-        <a
+        <button
           onClick={e => handleClick(e)}
           href="#"
```

Comment:
We can probably remove the `href` attribute here since we're changing this to a button element

Reply:
good call, I have this same change in two PRs, one of them I removed it and one I missed it :)